src/main/java/brokeculator/command/AddEventCommand.java

```diff
@@ -0,0 +1,25 @@
+package brokeculator.command;
+
+import brokeculator.dashboard.Dashboard;
+import brokeculator.event.Event;
+import brokeculator.frontend.UI;
+
+public class AddEventCommand extends Command {
+    
+    private String eventName;
+    private String eventDescription;
+
+    public AddEventCommand(String eventName, String eventDescription) {
+        assert eventName != null : "Event name cannot be null";
+        assert eventDescription != null : "Event description cannot be null";
+        this.eventName = eventName;
+        this.eventDescription = eventDescription;
+    }
+
+    @Override
+    public void execute(Dashboard dashboard) {
```

Comment:
Should adding an expense to an event remove it from the "general list" since it is now under a specific event? Worth considering.